checker/tests/nullness/ExceptionParam.java

```diff
@@ -14,14 +14,15 @@ void exc1() {
 
     void exc2() {
         try {
+            // :: warning: (nullness.on.exception.parameter)
         } catch (@NonNull AssertionError e) {
             @NonNull Object o = e;
         }
     }
 
     void exc3() {
         try {
-            // :: error: (type.invalid.annotations.on.use)
+            // :: warning: (nullness.on.exception.parameter)
```

Comment:
Should this be an error?  The other `nullness.on.*` messages are reported as errors.

Reply:
This one is intentionally a warning, but the explanation deserves a comment.  I will add one.